alpharaw/match/psm_match.py

```diff
@@ -30,6 +30,25 @@
 class PepSpecMatch:
     """
     Extract fragment ions from MS2 data.
+
+    Parameters
```

Comment:
hm, is there a reason why you moved that from the `__init__()` method? I feel it would be "closer to the code" there, i.e. more likely to be changed/adapted if the code also changes ..

Reply:
Yes

Reply:
I will use init for all classes